src/containers/Flow/FlowList/FlowList.tsx

```diff
@@ -68,6 +74,12 @@ export const FlowList: React.SFC<FlowListProps> = () => {
   const [flowName, setFlowName] = useState('');
   const [importing, setImporting] = useState(false);
 
+  const [releaseFlow] = useLazyQuery(RELEASE_FLOW);
+
+  useEffect(() => {
+    releaseFlow();
```

Comment:
Any specific reason why we are always calling releaseFlow() on the listing screen?

Reply:
We have many options on the floweditor screen that can be used to release flow namely:

1. After publishing
2. On click of the `back` button
3. On Click of `save as draft` button
4. When we close the tab

So rather than adding release flow on each of these actions, I thought this would be a much more optimal way. Let me know your thoughts.

Reply:
ok, that's fine.